pkg/cmd/cli/cmd/build.go

```diff
@@ -161,6 +161,8 @@ $ s2i build . centos/ruby-22-centos7 hello-world-app
 				}
 			}
 
+			manageConfigImageLabelsBuildImageName(cfg)
```

Comment:
Will need to create and pass in a context with my recommended refactoring

Reply:
are you fine with creating a `context.Background()` above this and then passing it into `manageConfigImageLabelsBuildImageName` @adambkaplan ?

Reply:
That would work